bin/publish-diff.js

```diff
@@ -16,11 +16,14 @@ pubDiff(opts, function (err, results) {
     // Convert to just text values
     .map(function (k) { return results.diff[k]; })
     // Keep non-empty diffs
-    .filter(function (d) { return d; })
-    // Colorize
-    .map(colorDiff)
-    // Convert to string
-    .join("\n");
+    .filter(function (d) { return d; });
+
+  // Colorize output if the user didn't pass --no-colors
+  if (opts.colors) {
+    output = output.map(colorDiff);
+  }
+  // Convert to a string
+  output = output.join("\n");
```

Comment:
Haha, I actually had the exact same thing, `identity` function and all, but rebased it to avoid calling `map` unnecessarily.

I can just revert if you'd prefer to keep it functional  👍